### PR TITLE
feat(arm): add CKV_AZURE_75 data explorer double encryption enabled convert policy to arm

### PR DESCRIPTION
**Pull request by goldyBH**
We converted the check AzureDataExplorerDoubleEncryptionEnabled from TERRAFORM language to the ARM language so that it also works on resources that are defined in the ARM language

Ensure that Azure Data Explorer uses double encryption
    Allowed prefixs:

    feat(arm): add azure data explorer double encryption enabled convert policy to arm
"

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from checkov.common.models.enums import CheckCategories
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+
+
+class AzureDataExplorerDoubleEncryptionEnabled(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure that Azure Data Explorer uses double encryption"
+        id = "CKV_AZURE_75"
+        supported_resources = ("Microsoft.Compute/disks",)
+        categories = [CheckCategories.ENCRYPTION]
```
Comment (ChanochShayner) [inline checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py:11]: Please use tuple instead of a list

**Review by ChanochShayner — APPROVED** [review]
Looks good!
Please fix the failing jobs (Mypy, etc..)

**Comment by ChanochShayner** [comment]
@goldyBH Any reason why you closed this PR?

File: checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class AzureDataExplorerDoubleEncryptionEnabled(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name: str = "Ensure that Azure Data Explorer uses double encryption"
+        id: str = "CKV_AZURE_75"
+        supported_resources: tuple = ("Microsoft.Compute/disks",)
+        categories: tuple = (CheckCategories.ENCRYPTION,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self) -> str:
+        return "properties/doubleEncryptionEnabled"
```
Comment (ChanochShayner) [inline checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py:15]:
```suggestion
        return "properties/enableDoubleEncryption"
```

File: checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from typing import Any
+from checkov.common.models.enums import CheckCategories
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+
+
+class AzureDataExplorerDoubleEncryptionEnabled(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name: str = "Ensure that Azure Data Explorer uses double encryption"
+        id: str = "CKV_AZURE_75"
+        supported_resources: tuple = ("Microsoft.Compute/disks",)
```
Comment (ChanochShayner) [inline checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py:10]:
```suggestion
        supported_resources = ("Microsoft.Kusto/clusters",)
```

File: checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        categories: tuple = (CheckCategories.ENCRYPTION,)
```
Comment (ChanochShayner) [inline checkov/arm/checks/resource/AzureDataExplorerDoubleEncryptionEnabled.py:11]:
```suggestion
        categories = (CheckCategories.ENCRYPTION,)
```

File: tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/fail.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+      "name": "fail",
+      "sku": {
+        "name": "B_Gen5_2",
+        "size": "5120"
+      },
+      "properties": {
+        "version": "10.3",
+        "administratorLogin": "admin",
+        "administratorLoginPassword": "admin123",
+        "doubleEncryptionEnabled": false,
```
Comment (ChanochShayner) [inline tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/fail.json:18]:
```suggestion
        "enableDoubleEncryption": false,
```

File: tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/pass.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+      "name": "pass",
+      "sku": {
+        "name": "B_Gen5_2",
+        "size": "5120"
+      },
+      "properties": {
+        "version": "10.3",
+        "administratorLogin": "admin",
+        "administratorLoginPassword": "admin123",
+        "doubleEncryptionEnabled": true,
```
Comment (ChanochShayner) [inline tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/pass.json:18]:
```suggestion
        "enableDoubleEncryption": true,
```

File: tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/fail.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "apiVersion": "2018-06-01",
+      "type": "Microsoft.Compute/disks",
```
Comment (ChanochShayner) [inline tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/fail.json:7]:
```suggestion
      "type": "Microsoft.Kusto/clusters",
```

File: tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/pass.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "resources": [
+    {
+      "apiVersion": "2018-06-01",
+      "type": "Microsoft.Compute/disks",
```
Comment (ChanochShayner) [inline tests/arm/checks/resource/example_AzureDataExplorerDoubleEncryptionEnabled/pass.json:7]:
```suggestion
      "type": "Microsoft.Kusto/clusters",
```

File: tests/arm/checks/resource/test_AzureDataExplorerDoubleEncryptionEnabled.py [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+from checkov.runner_filter import RunnerFilter
+
+
+class TestAzureDataExplorerDoubleEncryptionEnabled(unittest.TestCase):
+    def test_summary(self):
+        test_files_dir = Path(__file__).parent / "example_AzureDataExplorerDoubleEncryptionEnabled"
+        report = Runner().run(root_folder=str(test_files_dir), runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+        passing_resources = {
+            "Microsoft.Compute/disks.pass"
```
Comment (ChanochShayner) [inline tests/arm/checks/resource/test_AzureDataExplorerDoubleEncryptionEnabled.py:14]:
```suggestion
            "Microsoft.Kusto/clusters.pass"
```

File: tests/arm/checks/resource/test_AzureDataExplorerDoubleEncryptionEnabled.py [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+        }
+        failing_resources = {
+            "Microsoft.Compute/disks.fail"
```
Comment (ChanochShayner) [inline tests/arm/checks/resource/test_AzureDataExplorerDoubleEncryptionEnabled.py:17]:
```suggestion
            "Microsoft.Kusto/clusters"
```

File: tests/arm/checks/resource/test_AzureDataExplorerDoubleEncryptionEnabled.py [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+class TestAzureDataExplorerDoubleEncryptionEnabled(unittest.TestCase):
+    def test_summary(self):
+        test_files_dir = Path(__file__).parent / "example_AzureDataExplorerDoubleEncryptionEnabled"
+        report = Runner().run(root_folder=str(test_files_dir), runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+        passing_resources = {
+            "Microsoft.Kusto/clusters.pass"
+        }
+        failing_resources = {
+            "Microsoft.Kusto/clusters"
```
Comment (ChanochShayner) [inline tests/arm/checks/resource/test_AzureDataExplorerDoubleEncryptionEnabled.py:17]:
```suggestion
            "Microsoft.Kusto/clusters.fail"
```

**Review by SteveVaknin — APPROVED** [review]

**Pull request merged**